tests/AssetManagerTest/Resolver/AggregateResolverTest.php

```diff
@@ -8,6 +8,11 @@
 
 class AggregateResolverTest extends PHPUnit_Framework_TestCase
 {
+    public function setUp()
+    {
+        require_once __DIR__ . '/../../_files/ResolverWithCollect.php';
```

Comment:
Do you want me to change [AssetManagerTest\Service\AggregateResolverServiceFactoryTest](https://github.com/RWOverdijk/AssetManager/blob/master/tests/AssetManagerTest/Service/AggregateResolverServiceFactoryTest.php#L18) as well?